cmd/vcert/result_writer.go

```diff
@@ -205,33 +206,57 @@ func (r *Result) Flush() error {
 			if r.Config.ChainFile == "" {
 				certFileOutput.Chain = r.Pcc.Chain
 			}
-			bytes, err := certFileOutput.Format(r.Config)
-			if err != nil {
-				return err // something worse than file permission problem
-			} else {
-				err = ioutil.WriteFile(r.Config.CertFile, bytes, 0600)
-				errors = append(errors, err)
-			}
+			err = writeFile(certFileOutput, r.Config, r.Config.CertFile)
+			//bytes, err := certFileOutput.Format(r.Config)
+			//if err != nil {
+			//	return err // something worse than file permission problem
+			//} else {
+			//	err = ioutil.WriteFile(r.Config.CertFile, bytes, 0600)
```

Comment:
Let's remove old code rather than leaving it commented out

Reply:
Removed unneeded code